### PR TITLE
Add CircleCI

### PR DESCRIPTION
**Pull request by SviridovSV**

### CODE REVIEW

**Comment by SviridovSV** [comment]
@andriyParashchuk  Please check

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        environment:
+          RAILS_ENV: test
+          RACK_ENV: test
+          PGHOST: 127.0.0.1
+          PGUSER: ubuntu
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      - image: cockroachdb/cockroach:v2.1.3
```
Comment (andriyParashchuk) [inline .circleci/config.yml]: We used cockroachdb?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,126 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+# https://circleci.com/docs/2.0/configuration-reference
+version: 2
+jobs:
+  test:
+    environment:
+      BUNDLE_GEMFILE: Gemfile
+      BUNDLE_WITHOUT: "production:staging:development"
```
Comment (andriyParashchuk) [inline .circleci/config.yml]: It's need?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+          command: bundle check --path vendor/bundle || bundle install --deployment --jobs=4 --retry=3
+
+      - save_cache:
+          paths:
+            - ./vendor/bundle
+          key: dependencies-v4-{{ checksum "Gemfile.lock" }}
+
+      - restore_cache:
+          keys:
+            - yarn-packages-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
```
Comment (andriyParashchuk) [inline .circleci/config.yml]: ?

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+workflows:
+  version: 2
+  release:
+    jobs:
+      - test
+      - lint:
+          filters:
+            branches:
+              ignore: /^(master|production|staging)$/
```
Comment (andriyParashchuk) [inline .circleci/config.yml]: staging?

**Pull request merged**